index.js

```diff
@@ -0,0 +1,142 @@
+'use strict'
+
+if (!WebAssembly) {
+    throw new Error('WebAssembly not supported here.');
+}
+
+var loadModule = require('./generated/argon2.js');
+var wasmBinary = require('./generated/argon2.wasm.js');
+
+// XX this is unused -- should it be set in Module?
+
+// const KB = 1024 * 1024;
+// const MB = 1024 * KB;
+// const GB = 1024 * MB;
+// const WASM_PAGE_SIZE = 64 * 1024;
+
+// const totalMemory = (2*GB - 64*KB) / 1024 / WASM_PAGE_SIZE;
+// const mem = args.mem || +(1024);
+// const initialMemory = Math.min(Math.max(Math.ceil(mem * 1024 / WASM_PAGE_SIZE), 256) + 256, totalMemory);
+// const wasmMemory = new WebAssembly.Memory({
+//     initial: initialMemory,
+//     maximum: totalMemory
+// });
```

Comment:
@Fang-, were these settings supposed to be passed to the emscripten module runner? Or do the memory arguments to the emscripten build suffice?

Reply:
/shrugs  
The original [passes this in as part of the `Module`](https://github.com/antelle/argon2-browser/blob/master/docs/js/calc.js#L96), but if this works without that here then maybe this is deprecated and/or taken care of for us by emscripten.